.devcontainer/README.md

```diff
@@ -0,0 +1,7 @@
+# Overview
+
+Do not use a default configuration when starting a new devcontainer.
+Peek one of provided.
```

Comment:
What is meant here?

Reply:
We prebuild the codespace containers

Reply:
I mean with `Peek one of provided`.

Reply:
In UI, when there are 2 devconianers, in GitHub you see button start devcontaner, but it will start default something. While our containers are in submenu. Will update exact butto click here. UX is not good.